ksqldb-serde/src/main/java/io/confluent/ksql/serde/GenericKeySerDe.java

```diff
@@ -139,17 +139,24 @@ public GenericKeySerDe() {
     return serde;
   }
 
-  private static boolean unsupportedSchema(final List<SimpleColumn> columns) {
-    if (columns.isEmpty()) {
-      return false;
-    }
+  private static boolean unsupportedSchema(
```

Comment:
I was scratching my head trying to figure out why in the world we would return true if we had more than one column, then I realized it was named `unsupportedSchema` haha...